modules/misunderstood/src/views/full/MainScreen.tsx

```diff
@@ -0,0 +1,300 @@
+import { Button, ButtonGroup, HTMLTable, Intent } from '@blueprintjs/core'
+import clsx from 'clsx'
+import React from 'react'
+
+import {
+  ApiFlaggedEvent,
+  ContextMessage,
+  DbFlaggedEvent,
+  FLAGGED_MESSAGE_STATUS,
+  RESOLUTION_TYPE,
+  ResolutionData
+} from '../../types'
+
+import style from './style.scss'
+import { RESOLUTION } from './util'
+
+const DeletedList = ({
+  events,
+  totalEventsCount,
+  undeleteEvent
+}: {
+  events: DbFlaggedEvent[]
+  totalEventsCount: number
+  undeleteEvent: (id: string) => void
+}) => (
+  <>
+    <h3>Ignored Misunderstood ({totalEventsCount})</h3>
+    {events && !!events.length && (
+      <HTMLTable condensed interactive striped>
+        <thead>
+          <tr>
+            <td>Phrase</td>
+            <td>Deleted</td>
+            <td>&nbsp;</td>
+          </tr>
+        </thead>
+        <tbody>
+          {events.map((event, i) => (
+            <tr key={i}>
+              <td>{event.preview}</td>
+              <td>{event.updatedAt}</td>
+              <td>
+                <Button onClick={() => undeleteEvent('' + event.id)} small icon="refresh" intent={Intent.PRIMARY}>
+                  Restore
+                </Button>
```

Comment:
Good thing to know: Blueprint button has a property "text" that you can set instead of having the text as a children

Reply:
OK though I actually find it more readable this way, and closer to both HTML and other UI frameworks I've used